### PR TITLE
deps: use celestia-node v0.6.1, celestia-app v0.11.0

### LINKED ISSUE #37: Update dependencies
Use following version of docker images (to ensure Arabica testnet compatibility):

- celestia-node v0.6.1
- celestia-app v0.11.0

### PR DESCRIPTION
**Pull request by tzdybal**
## Overview

Resolves: #37 
Blocked-by: #40 

## Checklist

- [x] New and updated code has appropriate documentation
- [x] New and updated code has new and/or updated testing
- [x] Required CI checks are passing
- [x] Visual proof for any user facing features like CLI or documentation updates
- [x] Linked issues closed with keywords

### CODE REVIEW

File: integration_test/docker/test-docker-compose.yml [hunk 1]
```diff
@@ -3,7 +3,7 @@ version: '3'
 services:
   core0:
     container_name: core0
-    image: "ghcr.io/celestiaorg/celestia-app:v0.10.0-rc1"
+    image: "ghcr.io/celestiaorg/celestia-app:v0.10.0"
```
Comment (nashqueue) [inline integration_test/docker/test-docker-compose.yml:6]: Regarding Issue #37 Should this be Version 10 or 11 ?

**Review by nashqueue — CHANGES_REQUESTED** [review]

**Review by tuxcanfly — DISMISSED** [review]
LGTM.

File: integration_test/docker/test-docker-compose.yml (hunk 1 above)
Comment (nashqueue) [reply integration_test/docker/test-docker-compose.yml:6]: works with 0.10.0, error 13 with 0.11.0 , so I changed Title

**Review by nashqueue — DISMISSED** [review]

File: integration_test/docker/test-docker-compose.yml (hunk 1 above)
Comment (tzdybal) [reply integration_test/docker/test-docker-compose.yml]: We need v0.11.0 compatibility, as this is version used by Arabica testnet.

**Review by nashqueue — APPROVED** [review]

**Review by Manav-Aggarwal — APPROVED** [review]

**Pull request merged**